### ISSUE TITLE
Changable size of touchable area of the thumbImage

### ISSUE THREAD

**Issue by trashcoder** [react-native-community/react-native-slider#227]
## Describe the Feature

With the possibility to add a thumb image, it would be good to make the whole image touchable. Actually you can only touch a little area in the center of the image.

## Possible Implementations

Just like react-native-slider it could be implemented in props as "thumbTouchSize".

**Comment by jawwad22** [comment]
you can acheive this feature in this way	
```
style={ {
																		transform: [{ scaleX: .75 }, { scaleY: .75 }, ) }]
								
								} }
```

**Comment by trashcoder** [comment]
Thank you, but while scaleY works, scaleX does nothing:
![image](https://user-images.githubusercontent.com/2404165/102068498-c7519e80-3dfc-11eb-970e-372b17938a0e.png)

**Comment by endruuu** [comment]
Yeah, `scaleX` doesn't work if we wrap the slider inside a class component. But, it works within a functional component.
I tried on version 4.1.10.
